src/platform/user/profile/vet360/components/MobilePhone.jsx

```diff
@@ -1,22 +1,36 @@
 import React from 'react';
-import environment from 'platform/utilities/environment';
+import { connect } from 'react-redux';
 import PhoneField from './PhoneField';
 import ReceiveTextMessages from '../containers/ReceiveTextMessages';
 import { FIELD_NAMES } from '../constants';
+import { profileShowNotifications } from '../selectors';
 
-export default function MobilePhone() {
-  return (
-    <>
-      <PhoneField
-        title="Mobile phone number"
-        fieldName={FIELD_NAMES.MOBILE_PHONE}
-      />
-      {!environment.isProduction() && (
-        <ReceiveTextMessages
+class MobilePhone extends React.Component {
+  render() {
+    return (
+      <>
+        <PhoneField
           title="Mobile phone number"
           fieldName={FIELD_NAMES.MOBILE_PHONE}
         />
-      )}
-    </>
-  );
+        {this.props.showNotifications && (
```

Comment:
...and then remove the `this` from in front of `props`. I don't feel super strongly about this, but it seems better to keep it a functional component since it's not using any lifecycle methods.

Reply:
Thanks!